examples/zkapps/anonymous-message-board/src/message.ts

```diff
@@ -0,0 +1,77 @@
+import {
+  Field,
+  SmartContract,
+  state,
+  State,
+  method,
+  PrivateKey,
+  PublicKey,
+  Poseidon,
+} from 'o1js';
+  
+  // These private keys are exported so that experimenting with the contract is
```

Comment:
indentation still seems off for the rest of the file, I think we should fix that (make sure to install prettier and setup prettier on-save in vscode) https://www.robinwieruch.de/how-to-use-prettier-vscode/